conda/recipes/cuml/meta.yaml

```diff
@@ -41,6 +41,7 @@ requirements:
     - libcumlprims {{ minor_version }}
     - cupy>=6.5,<7.0
     - nccl 2.4.*
+    - conda-forge/label/rc_ucx::ucx-py=0.2
```

Comment:
I think the channel needs to be added outside the recipe.

Reply:
I was thinking this too, just because I've seen it done that way in our conda enviornment files

Reply:
conda-forge is outside the recipe, and we've used labels like these in the past: https://github.com/rapidsai/cuml/blob/0b320f025316a6fa7a498cd44b60ec92d115a9b5/conda/environments/cuml_dev_cuda9.2.yml#L12 @jakirkham so you're saying that the channel+label needs to be in the condarc as well?

Reply:
I'm not aware of conda-build supporting this syntax in recipes (though agree it would be useful). Environment files are a different story.

Or at least it needs to be in the install and build commands that are called. Though the `.condarc` would be better if that is a reasonable option.

Reply:
@jakirkham, are we still aiming to cut an official ucx-py conda-forge release for RAPIDS 0.11?

Reply:
If the schedules align, we might just be able to run with the official conda-forge version rather than using the rc labels. Or does this seem like it's side-stepping the problem?

Reply:
It's more likely we have a catch-22. 😅 Namely we want a proper release from upstream, but we've also identified a few more fixes needed as a result of trying it.

IMHO the sooner we can test the package here the better.

Reply:
@cjnolet I don't think it would be a big issue to add the channel to the `.condarc`, it is here if I'm not mistaken: https://github.com/rapidsai/gpuci-build-environment/blob/master/.condarc If both you and @jakirkham agree to add the RC package before or just after the issues with cuml.dask are resolved, we can just move to push for it being added to the condarc.